docs/en/how-to-loom-js-webpack.md

```diff
@@ -0,0 +1,365 @@
+  devtool: 'inline-source-map',
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+}
+```
+
+## The Webpack development server
+
+1. Firt, we must install the Webpack development server with:
+
+```js
+npm install --save-dev webpack-dev-server
+```
+
+2. Now that the required package is installed, let's enable the Webpack development server by adding the following lines to the `module.exports` section of the `webpack.config.js` file:
+
+```js
+devServer: {
+  contentBase: './dist',
+},
+
+```
+
+3. For convenience, we would want to start the web server using `npm run start`. To do so, add the following line to the `scripts` section of the `package.json` file:
+
+```js
+"start": "webpack-dev-server --open"
+```
+
+## Tweaking the webpack.config.js file
+
+1. Because we want to run this in the browser, we must add the following snippet to the `module.exports` section of the `webpack.config.js` file:
+
+```js
+node: {
+ fs: 'empty',
+ child_process: 'empty'
+}
+```
+
+This way, we'll get rid of the compilation errors.
+
+## Babel
+
+[Babel](https://babeljs.io/docs/en/index.html) is a tool that converts ECMAScript 2015+ code into JavaScript code that older browsers can run. With Babel, you can use the latest syntax without having to worry that a particular browser can't run it yet.
+
+1. Enter the following command to install Babel as a development dependency:
+
+```bash
+npm install --save-dev @babel/core @babel/cli @babel/preset-env babel-loader
+```
+
+2. To use Babel with Webpack, all what's left to do is to add the following snippet to your `webpack.config.js`:
+
+
+```js
+module: {
+  rules: [
+    {
+      test: [/.js$/],
+      exclude: /(node_modules)/,
+      use: {
+        loader: 'babel-loader',
+        options: {
+          presets: [
+            '@babel/preset-env'
+          ]
+        }
+      }
+    },
+  ]
+},
+```
+
+At this point, your `webpack.config.js` file should look something like the following:
+
+```js
+
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  devtool: 'inline-source-map',
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  devServer: {
+    contentBase: './dist',
+  },
+  node: {
+    fs: 'empty',
+    child_process: 'empty'
+  },
+  module: {
+    rules: [
+      {
+        test: [/.js$/],
+        exclude: /(node_modules)/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: [
+              '@babel/preset-env'
+            ]
+          }
+        }
+      },
+    ]
+  },
+}
+```
+
+## The Demo
+
+For the scope of this tutorial, we'll write a simple demo that deploys a contract to our testnet, writes a value to the blockchain and then reads it.
+
+1. First, let's install a few dependencies:
+
+```bash
+npm install loom-js web3 solcjs-core
+```
+
+
+2. Next, edit your `src/index.js` to this:
```

Comment:
What does `edit` mean here, does the file exist already and they have to replace the entire contents, or do they need to create a new one containing this code?